core/app/modules/import-export/iterator.php

```diff
@@ -46,10 +46,18 @@ public function __construct( array $settings ) {
 
 		$server = new Server();
 
-		$server_write_permissions = $server->get_write_permissions();
-
-		if ( $server_write_permissions['warning'] ) {
-			throw new \Error( self::NO_WRITE_PERMISSIONS_KEY );
+		$paths_to_check = [
+			$server::KEY_PATH_WP_CONTENT_DIR,
```

Comment:
```suggestion
			Server::KEY_PATH_WP_CONTENT_DIR,
```

easier to jump to the class

Reply:
Done